.github/scripts/generate_binary_build_matrix.py

```diff
@@ -209,6 +209,30 @@ def generate_wheels_matrix(os: str,
             # Skip rocm 3.11 binaries for now as the docker image are not correct
             if python_version == "3.11" and gpu_arch_type == "rocm":
                 continue
+
+            # special 11.7 wheels package without dependencies
+            # dependency downloaded via pip install
+            if arch_version == "11.7":
+                ret.append(
+                    {
+                        "python_version": python_version,
+                        "gpu_arch_type": gpu_arch_type,
+                        "gpu_arch_version": gpu_arch_version,
+                        "desired_cuda": translate_desired_cuda(
+                            gpu_arch_type, gpu_arch_version
+                        ),
+                        "container_image": WHEEL_CONTAINER_IMAGES[arch_version],
+                        "package_type": package_type,
+                        "pytorch_extra_install_requirements":
+                        "nvidia-cuda-runtime-cu11, nvidia-cudnn-cu11, nvidia-cublas-cu11",
```

Comment:
Please pin versions (and also use semicolon as separator)
```suggestion
                        "nvidia-cuda-runtime-cu11; nvidia-cudnn-cu11==8.5.0; nvidia-cublas-cu11==11.10.3",
```